external/fv3fit/fv3fit/tfdataset.py

```diff
@@ -29,6 +29,13 @@ def apply_to_tuple(
     return tuple(tensor_func(tensor) for tensor in data)
 
 
+def sequence_size(seq):
```

Comment:
Maybe an expensive function if loading is triggered

Reply:
Unfortunately there's no other way to count tfdataset members. We already do this in the dense code.